neuralprophet/forecaster.py

```diff
@@ -811,6 +813,18 @@ def predict(self, df, decompose=True, raw=False):
             forecast = pd.concat((forecast, fcst), ignore_index=True)
         df = df_utils.return_df_in_original_format(forecast, received_ID_col, received_single_time_series)
         self.predict_steps = self.n_forecasts
+        # Conformal prediction interval with q
+        if self.q_hats:
```

Comment:
Can we use a configuration dataclass (or merge it with the QR class) to configure CP?

Reply:
I'll create a `ConformalPrediction` dataclass in configure.py, which will contain `q_hats`.